src/test/resources/regressions/features/defunc/mutex1.gobra

```diff
@@ -0,0 +1,67 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+// These examples were designed and proposed by Linard Arquint
+package pkg
+
+// Mutex definition
+type Mutex struct {
```

Comment:
we'll need to rewrite this one as soon as we have the mutex stub but leave it for now

Reply:
Won't that be true only if we import the `sync` package which is not done in this test?

Reply:
True but that would make imho sense

Reply:
sure, let's change this then once we add the mutex stubs